delegation-verify-coordinator/values.yaml

```diff
@@ -0,0 +1,101 @@
+# Default values for delegation-verify-coordinator.
+replicas: 1
+
+image:
+  repository: ""
+  pullPolicy: IfNotPresent
+  tag: ""
+
+nameOverride:
+fullnameOverride:
+
+coordinator:
+  networkName: ""
+  platform: ""
+  cqlsh: ""
+  surveyIntervalMinutes: 20
+  command:
+    override: # use '"<command>"' notation to template correctly
+  postgres:
+    host: '{{ .Release.Name }}-postgresql'
+    port: 5432
+    db: ""
+    user: ""
+    password: ""
+  miniBatchNumber: 2
+  uptimeDaysForScore: ""
+  worker:
+    image: ""
+    tag: ""
+    cpu:
+      request: "3"
+      limit: "4"
+    memory:
+      request: "3072Mi"
+      limit: "4096Mi"
+  aws:
+    host: "cassandra.us-west-2.amazonaws.com"
+    port: "9142"
+    keyspace: ""
+    region: us-west-2
+    accessKeyID:
+    secretAccessKey:
+    s3Bucket: "673156464838-uptime-service-backend"
+  ssl:
+    certfile:
+
+serviceAccount:
+  coordinator:
+
+    # Annotations to add to the service account
+    annotations: {}
+
+  worker:
+
+    # Annotations to add to the service account
+    annotations: {}
+     
+  # Specifies whether a service account should be created
+  create: true
+  # Automatically mount a ServiceAccount's API credentials?
+  automount: true
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+podLabels: {}
+affinity: {}
+
+service:
+  type: ClusterIP
+  port: 80
+
+resources:
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  limits:
+    cpu: 100m
+    memory: 128Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
```

Comment:
The default Helm convention suggest not to put any values here
```
resources: {}
```
We will adjust them on the helmfile